### PR TITLE
[EDU-3829] feat: add manage objects in buckets guide

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Separate guides from #576 into this guide

### Additional links

Reference - Object Storage: #569
Reference - Runtime: #545
Guide - Manage object storage: #576

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EDU-3829]: https://aziontech.atlassian.net/browse/EDU-3829?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 1]
```diff
@@ -0,0 +1,159 @@
+---
+title: How to upload and download objects from an Azion Object Storage bucket
+description: Learn how to upload and download files from your object storage.
+meta_tags: object storage, storage, cloud, s3, bucket, file storage
+namespace: documentation_products_object_storage_upload
+permalink: /documentation/products/guides/how-to-upload-and-download-objects-from-bucket/
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:6]:
```suggestion
permalink: /documentation/products/guides/upload-and-download-objects-from-bucket/
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 2]
```diff
@@ -0,0 +1,161 @@
+---
+title: Como fazer o upload e download de objetos de um bucket do Azion Object Storage
+description: Aprenda como fazer upload e download de arquivos de buckets do Azion Object Storage.
+meta_tags: object storage, storage, cloud, s3, bucket, file storage
+namespace: documentation_products_object_storage_upload
+permalink: /documentacao/produtos/guias/como-fazer-upload-e-download-de-objetos-do-bucket/
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:6]:
```suggestion
permalink: /documentacao/produtos/guias/fazer-upload-e-download-de-objetos-do-bucket/
```

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+---
+
+import Button from '~/components/Button.astro'
+
+**Object Storage** allows you you to create buckets and integrate your object storage with your edge infrastructure. This guide covers how to upload and download objects from a bucket using the [Azion API](https://api.azion.com) and [Edge Runtime](/en/documentation/devtools/runtime/overview/).
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:11]:
```suggestion
**Azion Object Storage** allows you you to create buckets and integrate your object storage with your edge infrastructure. This guide covers how to upload and download objects from a bucket using the [Azion API](https://api.azion.com) and [Edge Runtime](/en/documentation/devtools/runtime/overview/).
```

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+permalink: /documentation/products/guides/upload-and-download-objects-from-bucket/
+---
+
+import Button from '~/components/Button.astro'
+
+**Object Storage** allows you you to create buckets and integrate your object storage with your edge infrastructure. This guide covers how to upload and download objects from a bucket using the [Azion API](https://api.azion.com) and [Edge Runtime](/en/documentation/devtools/runtime/overview/).
+
+<Button href="/en/documentation/products/store/object-storage/" text="go to Object Storage reference documentation" variant="secondary"></Button>
+
+Read [How to create and modify an Azion Object Storage bucket](/en/documentation/products/guides/how-to-upload-and-download-objects-from-bucket/) for bucket operations.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:15]: não seria o link do outro guia?

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+```json
+{
+    "state": "executed",
+    "data": {
+        "object_key": "folder/csv-file"
+    }
+}
+```
+
+### Download an object from a bucket
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:44]:
```suggestion
### Downloading an object from a bucket
```

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+---
+
+## Via Edge Runtime
+
+You can create an edge function to upload objects to your bucket using a `POST` console command and see if an object is available using a `GET` console command. To do so:
+
+1. Access [Real-Time Manager (RTM)](https://manager.azion.com/) > **Edge Functions**.
+2. Click the **Add Function** button.
+3. Name your function. Example: `my-bucket POST+GET`
+4. In the **Code** tab, add the following Javascript code:
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:63]:
```suggestion
4. In the **Code** tab, add the following JavaScript code:
```

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+    }
+}
+addEventListener("fetch", (event) => {
+    event.respondWith(
+        router(event)
+    );
+});
+```
+
+| Variable | Expected value |
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:112]: o label ficou Expect value mas tem description + examples

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+}
+```
+
+6. Click the **Save** button.
+
+:::note
+See the [Object Storage API](/en/documentation/devtools/runtime/api-reference/storage/) reference for more details on functions available for other types of requests.
+:::
+
+Once you have the edge function ready, you need to create an edge application that will proxy the upload process for the Object Storage.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:133]: instância do object storage, em minúsculo?

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+
+1. Access [Real-Time Manager (RTM)](https://manager.azion.com/)
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:135]:
```suggestion
1. Access [Real-Time Manager (RTM)](https://manager.azion.com/).
```

File: src/content/docs/pt-br/pages/guias/guides.mdx [hunk 10]
```diff
@@ -24,6 +24,7 @@ permalink: /documentacao/produtos/guias/
 - [Como gerar um certificado Let's Encrypt para o seu domínio](/pt-br/documentacao/produtos/guias/como-gerar-um-certificado-lets-encrypt/)
 - [Como criar listas de bloqueio de IP, ASN e geolocalização com Network Lists](/pt-br/documentacao/produtos/guias/blacklists-enderecos-ip-edge/)
 - [Como fazer o staging de uma edge application através do arquivo hosts](/pt-br/documentacao/produtos/guias/testar-edge-application-atraves-do-arquivo-hosts/)
+- [Como fazer o upload e download de objetos de um bucket do Azion Object Storage](/pt-br/documentacao/produtos/guias/como-fazer-upload-e-download-de-objetos-do-bucket/)
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/guides.mdx:27]:
```suggestion
- [Como fazer o upload e download de objetos de um bucket do Azion Object Storage](/pt-br/documentacao/produtos/guias/upload-e-download-de-objetos-do-bucket/)
```

File: src/content/docs/en/pages/guides/index.mdx [hunk 11]
```diff
@@ -23,6 +23,7 @@ permalink: /documentation/products/guides/
 - [How to create rules using Rules Engine for Edge Application](/en/documentation/products/guides/static-sites-on-edge/)
 - [How to create IP, ASN, and geolocation blocklists with Network Lists](/en/documentation/products/guides/blacklists-ip-addresses-edge/)
 - [How to stage an edge application through the hosts file](/en/documentation/products/guides/stage-edge-application-through-hosts-file/)
+- [How to upload and download objects from an Azion Object Storage bucket](/en/documentation/products/guides/how-to-upload-and-download-objects-from-bucket/)
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/index.mdx:26]:
```suggestion
- [How to upload and download objects from an Azion Object Storage bucket](/en/documentation/products/guides/upload-and-download-objects-from-bucket/)
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 12]
```diff
@@ -0,0 +1,161 @@
+---
+title: Como fazer o upload e download de objetos de um bucket do Azion Object Storage
+description: Aprenda como fazer upload e download de arquivos de buckets do Azion Object Storage.
+meta_tags: object storage, storage, cloud, s3, bucket, file storage
+namespace: documentation_products_object_storage_upload
+permalink: /documentacao/produtos/guias/fazer-upload-e-download-de-objetos-do-bucket/
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:6]:
```suggestion
permalink: /documentacao/produtos/guias/upload-e-download-de-objetos-do-bucket/
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 13]
```diff
@@ -0,0 +1,161 @@
...
+---
+
+import Button from '~/components/Button.astro'
+
+**Object Storage** permite que você crie buckets e integre seu armazenamento de objetos com sua infraestrutura do edge. Este guia abrange como fazer upload e download de objetos de um bucket usando a [API da Azion](https://api.azion.com) e [Edge Runtime](/pt-br/documentation/devtools/runtime/overview/).
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:11]:
```suggestion
**Azion Object Storage** permite que você crie buckets e integre seu armazenamento de objetos com sua infraestrutura do edge. Este guia abrange como fazer upload e download de objetos de um bucket usando a [API da Azion](https://api.azion.com) e [Edge Runtime](/pt-br/documentation/devtools/runtime/overview/).
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 14]
```diff
@@ -0,0 +1,161 @@
...
+
+<Button href="/pt-br/documentacao/produtos/store/object-storage/" text="vá para a documentação de referência de Object Storage" variant="secondary"></Button>
+
+Consulte o guia [Como criar e modificar um bucket de Object Storage da Azion](/pt-br/documentacao/produtos/guias/como-criar-e-modificar-um-bucket/) para operações em buckets.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:15]:
```suggestion
Consulte o guia [Como criar e modificar um bucket de Object Storage da Azion](/pt-br/documentacao/produtos/guias/criar-e-modificar-um-bucket/) para operações em buckets.
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 15]
```diff
@@ -0,0 +1,161 @@
...
+
+---
+
+## Via API
+
+### Fazer upload de objeto para um bucket
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:21]:
```suggestion
### Faça upload de objeto para um bucket
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 16]
```diff
@@ -0,0 +1,161 @@
...
+```json
+{
+    "state": "executed",
+    "data": {
+        "object_key": "folder/csv-file"
+    }
+}
+```
+
+## Fazer download de objeto de um bucket
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:44]:
```suggestion
## Faça download de objeto de um bucket
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 17]
```diff
@@ -0,0 +1,161 @@
...
+---
+
+## Via Edge Runtime
+
+Você pode criar uma edge function para fazer upload de arquivos para seu bucket usando um comando de console `POST` e verificar se um arquivo está disponível usando um comando de console `GET`. Para fazer isso:
+
+1. Accesse o [Real-Time Manager (RTM)](https://manager.azion.com/) > **Edge Functions**.
+2. Clique no botão **Add Function**.
+3. Dê um nome a sua função. Exemplo: `my-bucket POST+GET`
+4. Na aba **Code**, adicione o seguinte código Javascript:
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:63]:
```suggestion
4. Na aba **Code**, adicione o seguinte código JavaScript:
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 18]
```diff
@@ -0,0 +1,161 @@
...
+    }
+}
+addEventListener("fetch", (event) => {
+    event.respondWith(
+        router(event)
+    );
+});
+```
+
+| Variável | Valor esperado |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:112]: mesmo comentário do label que fiz em EN

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 19]
```diff
@@ -0,0 +1,161 @@
...
+```json
+{
+  "bucket": "my-bucket"
+}
+```
+
+6. Clique no botão **Save**.
+
+:::note
+Consulte a referência [Object Storage API](/en/documentation/devtools/runtime/api-reference/storage/) para mais detalhes nas funções disponíveis para outros tipos de requisições.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:130]:
```suggestion
Consulte a referência [Object Storage API](/en/documentation/devtools/runtime/api-reference/storage/) para mais detalhes sobre as funções disponíveis para outros tipos de requisições.
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 20]
```diff
@@ -0,0 +1,161 @@
...
+:::
+
+Uma vez que você tenha a edge function, você precisa criar uma edge application que vai intermediar o processo de upload para o Object Storage.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:133]:
```suggestion
Após criar a edge function, você precisa criar uma edge application que vai intermediar o processo de upload para o Object Storage.
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 21]
```diff
@@ -0,0 +1,161 @@
...
+
+1. Acesse o [Real-Time Manager (RTM)](https://manager.azion.com/)
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:135]:
```suggestion
1. Acesse o [Real-Time Manager (RTM)](https://manager.azion.com/).
```

**Review by gabriel-azion — APPROVED** [review]

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 22]
```diff
@@ -0,0 +1,159 @@
...
+permalink: /documentation/products/guides/upload-and-download-objects-from-bucket/
+---
+
+import Button from '~/components/Button.astro'
+
+**Azion Object Storage** allows you you to create buckets and integrate your object storage with your edge infrastructure. This guide covers how to upload and download objects from a bucket using the [Azion API](https://api.azion.com) and [Edge Runtime](/en/documentation/devtools/runtime/overview/).
+
+<Button href="/en/documentation/products/store/object-storage/" text="go to Object Storage reference documentation" variant="secondary"></Button>
+
+Read [How to create and modify an Azion Object Storage bucket](/en/documentation/products/guides/how-to-upload-and-download-objects-from-bucket/) for bucket operations.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:15]:
```suggestion
Read [How to create and modify an Azion Object Storage bucket](/en/documentation/products/guides/create-and-modify-bucket/) for bucket operations.
```

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 23]
```diff
@@ -0,0 +1,159 @@
...
+    }
+}
+addEventListener("fetch", (event) => {
+    event.respondWith(
+        router(event)
+    );
+});
+```
+
+| Variable | Expected value |
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:112]:
```suggestion
| Variable | Description |
```

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 24]
```diff
@@ -0,0 +1,159 @@
...
+}
+```
+
+6. Click the **Save** button.
+
+:::note
+See the [Object Storage API](/en/documentation/devtools/runtime/api-reference/storage/) reference for more details on functions available for other types of requests.
+:::
+
+Once you have the edge function ready, you need to create an edge application that will proxy the upload process for the Object Storage.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:133]:
```suggestion
Once you have the edge function ready, you need to create an edge application that will proxy the upload process for the bucket.
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 25]
```diff
@@ -0,0 +1,161 @@
...
+    }
+}
+addEventListener("fetch", (event) => {
+    event.respondWith(
+        router(event)
+    );
+});
+```
+
+| Variável | Valor esperado |
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:112]:
```suggestion
| Variável | Description |
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 26]
```diff
@@ -0,0 +1,161 @@
...
+}
+```
+
+6. Clique no botão **Save**.
+
+:::note
+Consulte a referência [Object Storage API](/en/documentation/devtools/runtime/api-reference/storage/) para mais detalhes sobre as funções disponíveis para outros tipos de requisições.
+:::
+
+Após criar a edge function, você precisa criar uma edge application que vai intermediar o processo de upload para o Object Storage.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:133]:
```suggestion
Após criar a edge function, você precisa criar uma edge application que vai intermediar o processo de upload para o bucket.
```

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 27]
```diff
@@ -0,0 +1,159 @@
...
+
+---
+
+## Via Edge Runtime
+
+You can create an edge function to upload objects to your bucket using a `POST` console command and see if an object is available using a `GET` console command. To do so:
+
+1. Access [Real-Time Manager (RTM)](https://manager.azion.com/) > **Edge Functions**.
+2. Click the **Add Function** button.
+3. Name your function. Example: `my-bucket POST+GET`
```
Comment (hannah-sk) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:62]:
```suggestion
3. Name your function. Example: `my-bucket POST+GET`.
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 28]
```diff
@@ -0,0 +1,161 @@
...
+
+---
+
+## Via Edge Runtime
+
+Você pode criar uma edge function para fazer upload de arquivos para seu bucket usando um comando de console `POST` e verificar se um arquivo está disponível usando um comando de console `GET`. Para fazer isso:
+
+1. Accesse o [Real-Time Manager (RTM)](https://manager.azion.com/) > **Edge Functions**.
+2. Clique no botão **Add Function**.
+3. Dê um nome a sua função. Exemplo: `my-bucket POST+GET`
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:62]:
```suggestion
3. Dê um nome a sua função. Exemplo: `my-bucket POST+GET`.
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 29]
```diff
@@ -0,0 +1,161 @@
...
+    }
+}
+addEventListener("fetch", (event) => {
+    event.respondWith(
+        router(event)
+    );
+});
+```
+
+| Variável | Description |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:112]:
```suggestion
| Variável | Descrição |
```

**Review by hannah-sk — APPROVED** [review]

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 30]
```diff
@@ -0,0 +1,159 @@
...
+title: How to upload and download objects from an Azion Object Storage bucket
+description: Learn how to upload and download files from your object storage.
+meta_tags: object storage, storage, cloud, s3, bucket, file storage
+namespace: documentation_products_object_storage_upload
+permalink: /documentation/products/guides/upload-and-download-objects-from-bucket/
+---
+
+import Button from '~/components/Button.astro'
+
+**Azion Object Storage** allows you you to create buckets and integrate your object storage with your edge infrastructure. This guide covers how to upload and download objects from a bucket using the [Azion API](https://api.azion.com) and [Edge Runtime](/en/documentation/devtools/runtime/overview/).
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:11]:
```suggestion
**Azion Object Storage** allows you to create buckets and integrate your object storage with your edge infrastructure. This guide covers how to upload and download objects from a bucket using the [Azion API](https://api.azion.com) and [Edge Runtime](/en/documentation/devtools/runtime/overview/).
```

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 31]
```diff
@@ -0,0 +1,159 @@
...
+--header 'Authorization: Token [TOKEN VALUE]'
+```
+
+---
+
+## Via Edge Runtime
+
+You can create an edge function to upload objects to your bucket using a `POST` console command and see if an object is available using a `GET` console command. To do so:
+
+1. Access [Real-Time Manager (RTM)](https://manager.azion.com/) > **Edge Functions**.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:60]:
```suggestion
1. Access [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/) > **Edge Functions**.
```

File: src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx [hunk 32]
```diff
@@ -0,0 +1,159 @@
...
+
+:::note
+See the [Object Storage API](/en/documentation/devtools/runtime/api-reference/storage/) reference for more details on functions available for other types of requests.
+:::
+
+Once you have the edge function ready, you need to create an edge application that will proxy the upload process for the bucket.
+
+1. Access [Real-Time Manager (RTM)](https://manager.azion.com/).
+2. Select the **Start with a template** option on the homepage.
+3. Select the option **Build from scratch**.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/object-storage/bucket-actions/index.mdx:137]:
```suggestion
3. Select the **Build from scratch** option.
```

File: src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx [hunk 33]
```diff
@@ -0,0 +1,161 @@
...
+--header 'Authorization: Token [TOKEN VALUE]'
+```
+
+---
+
+## Via Edge Runtime
+
+Você pode criar uma edge function para fazer upload de arquivos para seu bucket usando um comando de console `POST` e verificar se um arquivo está disponível usando um comando de console `GET`. Para fazer isso:
+
+1. Accesse o [Real-Time Manager (RTM)](https://manager.azion.com/) > **Edge Functions**.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/object-storage/bucket-actions/index.mdx:60]:
```suggestion
1. Accesse o [Real-Time Manager (RTM)](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) > **Edge Functions**.
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**